pkg/cover/backend/dwarf.go

```diff
@@ -261,8 +261,9 @@ func makeDWARFUnsafe(params *dwarfParams) (*Impl, error) {
 		Symbolize: func(pcs map[*Module][]uint64) ([]Frame, error) {
 			return symbolize(target, objDir, srcDir, buildDir, splitBuildDelimiters, pcs)
 		},
-		RestorePC:   makeRestorePC(params, pcBase),
-		CoverPoints: allCoverPointsMap,
+		RestorePC:              makeRestorePC(params, pcBase),
+		CallbackPoints:         allCoverPointsMap,
+		CoverageCallbackPoints: allCoverPoints[0],
```

Comment:
These are the additional PCs we will symbolize, right?
But isn't this the address of __sanitizer_cov_trace_pc function itself (which is not even a coverage point)?